.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote":true,
+    "printWidth":140,
+    "tabWidth":2

```

Comment:
Does tabWidth imply you're using tabs instead of spaces?

Reply:
Do you prefer spaces?